ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands2.java

```diff
@@ -2209,20 +2209,7 @@ public void testAcidOrcWritePreservesFieldNames() throws Exception {
    * sorts rows in dictionary order
    */
   static List<String> stringifyValues(int[][] rowsIn) {
-    assert rowsIn.length > 0;
-    int[][] rows = rowsIn.clone();
-    Arrays.sort(rows, new RowComp());
-    List<String> rs = new ArrayList<String>();
-    for(int[] row : rows) {
-      assert row.length > 0;
-      StringBuilder sb = new StringBuilder();
-      for(int value : row) {
-        sb.append(value).append("\t");
-      }
-      sb.setLength(sb.length() - 1);
-      rs.add(sb.toString());
-    }
-    return rs;
+    return TxnCommandsBaseForTests.stringifyValues(rowsIn);
```

Comment:
remove it and use static call

Reply:
I will do this in a separate Jira, started it, but it requires more change.